requirements.txt

```
@@ -8,7 +8,7 @@ matplotlib~=3.7
 mlflow~=2.3
 networkx~=3.1
 optuna~=3.1
-pandas==2.1.3
+pandas==2.2.0
```

```suggestion
pandas~=2.2.0
```
I would suggest using the compatible release operator, so that the package stays updated with low risks of breaking things, and allowing different versions in software that includes OpenSTEF.

---

👍 Nice suggestion. Fixed.